apps/web/app/(auth)/account/keys/page.tsx

```diff
@@ -0,0 +1,110 @@
+import { MaxWidthContainer } from "@/features/page/page";
+import { auth } from "@/lib/auth";
+import { getRequiredUser } from "@/lib/auth-session";
+import { serverToast } from "@/lib/server-toast";
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+} from "@workspace/ui/components/card";
+import { Typography } from "@workspace/ui/components/typography";
+import { revalidatePath } from "next/cache";
+import { headers } from "next/headers";
+import { ApiKeyList } from "./api-key-list";
+import { CreateApiKeyForm } from "./create-api-key-form";
+
+export default async function ApiKeysPage() {
+  const user = await getRequiredUser();
+
+  const createApiKey = async (formData: FormData) => {
```

Comment:
use server-action and form etc... like i did in other azth files like sign in component !